apps/minifront/src/components/swap/duration-slider.tsx

```diff
@@ -27,12 +27,16 @@ export const DurationSlider = () => {
         onValueChange={handleChange}
       />
 
-      <div className='flex flex-col items-center'>
-        ~ {duration}{' '}
-        <span className='text-xs text-muted-foreground'>
-          distributed across {GDA_RECIPES[duration].numberOfSubAuctions.toString()} sub-auctions
-        </span>
-      </div>
+      {duration === 'instant' && <>instant</>}
+
+      {duration !== 'instant' && (
```

Comment:
![image](https://github.com/penumbra-zone/web/assets/1121544/1d30e5cc-3114-45d3-8538-a5209ecaad59)

Reply:
Can we add "Instant Price" <> "Averaged Price" labels at either end of the slider?

Reply:
Ah, sure — by the way, this is part of why we need to discuss plans for implementing Sam's new Figma designs. This PR is only really intended to solve the problem of A) combining the two forms in the UI and B) combining the two forms in state. Obviously, I want to completely implement Sam's designs for this eventually, but they're using a new look and feel that we'll need to strategize how to implement throughout minifront/Prax.

Reply:
Done:
![image](https://github.com/penumbra-zone/web/assets/1121544/bf4d1e37-58cd-4521-b5c4-16051ce657ab)